src/status_im/utils/handlers.clj

```diff
@@ -0,0 +1,12 @@
+(ns status-im.utils.handlers)
+
+(defmacro handlers->
```

Comment:
do we really need this macro? because multiple handlers look tricky and confusing, maybe we can rewtrite them into pure functions and use inside one handler ?